airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -152,6 +152,8 @@ def read_records(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> Iter
                     f"Syncing `{self.name}` stream isn't available for repository "
                     f"`{stream_slice['repository']}`, it seems like this repository is empty."
                 )
+            elif e.response.status_code == requests.codes.SERVER_ERROR and isinstance(self, WorkflowRuns):
```

Comment:
does it make sense to skip server error independent of what current stream is?

Reply:
This fix is workaround for one specific case which I got on customer account.
It's pretty strange case and I could not understand why we got this 500 for only one repository.
As for me it looks like some internal bug on github side.